content/references/block-utils.md

```diff
@@ -54,6 +54,14 @@ This function adds a link to an external style sheet.
 utils.addLink('https://your-style-sheet.css')
 ```
 
+Volt will automatically optimize your CSS when rendering. This optimization can sometimes cause problems, such as with relative paths (including in `@import`s). To turn off the optimization, pass the `optimize: false` flag:
+
+```javascript
+utils.addLink('./your-style-sheet.css', { optimize: false })
```

Comment:
```suggestion
utils.addLink('https://www.example.com/your-style-sheet.css', { optimize: false });
```

Reply:
Would there be benefit to demonstrating that this works with relative paths? Maybe a prefix of `/` rather than `./` would be better given that blocks could be added to any page.

Reply:
I don't believe local CSS files are built and deployed, so no. You also don't have access to upload files to your domain on Volt, but you do on V1. I'd be ok with removing just the `.`